dimod/reference/samplers/exact_solver.py

```diff
@@ -73,30 +77,28 @@ def __init__(self):
         self.properties = {}
         self.parameters = {}
 
-    def sample(self, bqm, **kwargs):
+    def sample(self, bqm: BinaryQuadraticModel, **kwargs) -> SampleSet:
```

Comment:
To resolve the CI failures, you can using strings. e.g. `bqm: 'BinaryQuadraticModel'`